featuretools/tests/primitive_tests/natural_language_primitives_tests/test_natural_language_perf.py

```diff
@@ -0,0 +1,62 @@
+import signal
+from platform import system
+
+import pandas as pd
+import pytest
+
+from featuretools.primitives import (
+    CountString,
+    MeanCharactersPerWord,
+    MedianWordLength,
+    NumberOfHashtags,
+    NumberOfMentions,
+    NumberOfUniqueWords,
+    NumberOfWordsInQuotes,
+    NumUniqueSeparators,
+    PunctuationCount,
+    TitleWordCount,
+    TotalWordLength,
+    UpperCaseCount,
+    WhitespaceCount,
+)
+
+TIMEOUT_THRESHOLD = 20
+
+if system == "Linux" or system == "Darwin":
```

Comment:
Do Windows users skip these tests?

Reply:
I think the regular CI runs are skipping these tests as well.  The "unit tests core" job is reporting the same number of tests on main and this PR

Reply:
@ozzieD Yes. This is because `SIGALRM` is only available on UNIX systems.